### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by NeverovDS**
Function that returns the copy of the original
slice in reverse order.

### CODE REVIEW

File: reverseNumbers.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+)
+
+func max(sl []int64) []int64 {
```
Comment (burov) [inline reverseNumbers.go]: Why the name is max ?

File: reverseNumbers.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	k := make([]int64,0)
```
Comment (imatsiash-ib) [inline reverseNumbers.go]: the code is not `go fmt`-ed.

File: reverseNumbers.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import (
+	"fmt"
+)
+
+func reverseSlice(slice []int64) []int64 {
+	result := make([]int64, 0)
+
+	for i, j := int64(0), int64(len(slice)-1); i < j; i, j = i+1, j-1 {
+		slice[i], slice[j] = slice[j], slice[i]
+		result = append(slice)
```
Comment (imatsiash-ib) [inline reverseNumbers.go]: Please use the next form for the append: 
`func append(s []T, vs ...T) []T`

Comment (NeverovDS) [reply reverseNumbers.go]: without append))

File: reverseNumbers.go [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"fmt"
+)
+
+func reverseSlice(slice []int64) []int64 {
+	result := make([]int64, 0)
+
+	for i, j := int64(0), int64(len(slice)-1); i < j; i, j = i+1, j-1 {
+		slice[i], slice[j] = slice[j], slice[i]
```
Comment (imatsiash-ib) [inline reverseNumbers.go]: You should not change the input slice.

Comment (NeverovDS) [reply reverseNumbers.go]: it?

File: reverseNumbers.go [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+)
+
+func reverseSlice(slice []int64) []int64 {
+	result := make([]int64, 0)
+	result = slice
```
Comment (imatsiash-ib) [inline reverseNumbers.go]: Please use `copy` function instead.

Comment (NeverovDS) [reply reverseNumbers.go]: +

**Pull request closed without merging**